packages/quantic/force-app/main/default/lwc/quanticInsightInterface/quanticInsightInterface.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>53.0</apiVersion>
```

Comment:
Maybe the version should be 54.0 to be consistent with all the other components.